timeago/lib/src/timeago.dart

```diff
@@ -7,6 +8,8 @@ Map<String, LookupMessages> _lookupMessagesMap = {
   'en_short': EnShortMessages(),
   'es': EsMessages(),
   'es_short': EsShortMessages(),
+  'vi': ViMessages(),
```

Comment:
Let's not add it to the default locale map.

The issue is that adding them to the map will prevent them to be removed (tree shaken) if someone just uses for example english locale. So its better to only add the languages using `timeago.setLocaleMessages('vi', timeago.ViMessages());`

I probably should only have bundle english locale in that map, but removing spanish locale will make a breaking change.